content/applications/productivity/calendar.rst

```diff
@@ -9,3 +10,246 @@ Calendar
+confirmed a specified amount of time in advance. For example, select :guilabel:`01:00` to require
+attendees to confirm at least one hour before their appointment time.
+
+In the :guilabel:`Allow Cancelling` field, select a minimum hour window before the appointment, if
+attendees need to make a cancellation.
+
+The :guilabel:`Availability on` field enables attendees to book :guilabel:`Users` or
+:guilabel:`Resources`, such as meeting rooms or tables. After selecting
+:guilabel:`Users` or :guilabel:`Resources`, then type in the desired contact or resource in the
+space below.
+
+The :guilabel:`Front-End Display` field is used to choose :guilabel:`No Picture` or
+:guilabel:`Show Pictures` related to the selected user or resource on the appointment page.
+
+If :guilabel:`Resources` has been selected in the :guilabel:`Availability on` field, users have an
+option to :guilabel:`Manage Capacities`.
+
+Tick the checkbox to limit the maximum amount of people that can use the resource at the same time.
+
+The :guilabel:`Assignment Method` field enables the order in which attendees book their time and
+user/resource:
+
+- :guilabel:`Pick User/Resource then Time`
+- :guilabel:`Select Time then User/Resource`
+
+If :guilabel:`Resources` has been selected in the :guilabel:`Availability On` field, a third option
+is available, :guilabel:`Select Time then auto-assign`.
+
+Schedule
+~~~~~~~~
+
+In the :guilabel:`Schedule` tab of the new meeting page, add or delete time slots. The target date
+and time populate as the first time slots.
+
+To add a new time slot, hit :guilabel:`Add a line`. Click into the new blank space under the
+:guilabel:`From` field, then select and enter the new target start date and time, respectively.
+Repeat under the new blank space under :guilabel:`To` to select and enter the new target end date
+and time.
+
+.. image:: calendar/calendar-new-meeting.png
+   :align: center
+   :alt: New meeting page on Calendar app.
+
+Options
+~~~~~~~
+
+The :guilabel:`Options` tab provides additional configurations:
+
+- :guilabel:`Website`: Specify which website this meeting invitation will be published on.
+- :guilabel:`Timezone`: This is defaulted to the company's time zone selected in
+  the :menuselection:`Settings app`.
+  To change the time zone, click into the section, and select the desired option from the drop-down
+  menu.
+- :guilabel:`Location`: Select or create new locations from the drop-down menu. If this field is
+  left empty, the meeting is considered to be taking place online.
+- :guilabel:`Manual Confirmation`: If :guilabel:`Resources` has been selected in the
+  :guilabel:`Availability On` field, tick the checkbox and enter a maximum percentage of the
+  selected resource(s)' total capacity to create a manual confirmation requirement to finalize the
+  meeting.
+- :guilabel:`Videoconference Link`: Select from :guilabel:`Odoo Discuss` or :guilabel:`Google Meet`
+  to include a video conference link in the meeting invitation.
```

Comment:
Suggestion to swap the order of **Videoconference Link** and **Manual Confirmation** based on the order in the UI.

Also, specify that **Manual Confirmation** is only shown if _Resources_ is selected. And specify that the videoconference can be left blank to not generate a URL
```suggestion
- :guilabel:`Videoconference Link`: Select from :guilabel:`Odoo Discuss` or :guilabel:`Google Meet`
  to include a video conference link in the meeting invitation, or leave it blank to prevent
  generating a meeting URL.
- :guilabel:`Manual Confirmation`: Only shown if :guilabel:`Resources` has been selected in the
  :guilabel:`Availability On` field. Tick the checkbox and enter a maximum percentage of the
  selected resource(s)' total capacity to create a manual confirmation requirement to finalize the
  meeting.
```